.github/workflows/pre-release-fips.yml

```diff
@@ -0,0 +1,63 @@
+name: FIPS Pre-release
```

Comment:
Can we use the existing pre-release workflow and add the FIPS job to a strategy matrix as suggested for the build job?